src/lib/status/index.ts

```diff
@@ -0,0 +1,33 @@
+import axios from "axios";
+
+import { GELATO_RELAY_URL } from "../../constants";
+import { getHttpErrorMessage } from "../../utils";
+
+import { Service, TransactionStatus } from "./types";
+
+/**
+ * @param {string} taskId - Task Id
+ * @param {Service} [service] - Optional Service parameter: Service.Relay or Service.Relay_V2
+ * @returns {Promise<TransactionStatus | undefined>} Transaction status of the task id
+ *
+ */
+export const getTaskStatus = async (
+  taskId: string,
+  service?: Service
```

Comment:
no need to expose the `service` parameter
users of this SDK are only using RelayV2

Reply:
https://github.com/gelatodigital/relay-sdk/pull/5/commits/c3673510de14f5f41128e9e17d1fd9c2947ceb65